tests/BuiltInWebServerAbstract.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace atk4\ui\tests;
+
+use GuzzleHttp\Client;
+use PHPUnit\Framework\TestCase;
+use Psr\Http\Message\ResponseInterface;
+use Symfony\Component\Process\Process;
+
+abstract class BuiltInWebServerAbstract extends TestCase
```

Comment:
add alternative base Test class to test real GET POST request using built in server